crates/re_viewer_context/src/component_ui_registry.rs

```diff
@@ -313,37 +275,78 @@ impl ComponentUiRegistry {
     ///   (e.g. if you get a `Color` you can't assume whether it's a background color or a point color)
     /// * The returned [`egui::Response`] should be for the widget that has the tooltip, not any pop-up content.
     ///     * Make sure that changes are propagated via [`egui::Response::mark_changed`] if necessary.
-    pub fn add_multiline_editor_ui<C: re_types::Component>(
+    /// * It is fine to only implement a viewer with this, i.e. not handle the [`MaybeMutRef::MutRef`] case.
```

Comment:
one spot where this is a bit of a problem is the view defaults ui: We currently only add the option to add component defaults from ui that we can also edit. With this change, we'd offer it for some components that you can't actually edit. Not a huge problem but might be a bit annoying for established users and confusing for new ones 🤔